src/Microsoft.ML.Data/Utilities/ModelFileUtils.cs

```diff
@@ -66,9 +66,9 @@ public static IDataView LoadPipeline(IHostEnvironment env, RepositoryReader rep,
             env.CheckValue(files, nameof(files));
             using (var ent = rep.OpenEntry(DirDataLoaderModel, ModelLoadContext.ModelStreamName))
             {
-                IDataLoader loader;
+                ILegacyDataLoader loader;
```

Comment:
>ILegacyDataLoader [](start = 16, length = 17)

Just curious - will all code that uses the ILegacyDataLoader be also moved to legacy? Or does it need to be updated to use the DataLoader?